.github/ISSUE_TEMPLATE/cherry_pick_template.md

```diff
@@ -25,7 +25,7 @@ TIP: Cherry-picks into Stable most likely require a cherry-pick into Beta / Expe
 | :---------------: | :------------: | :------------------: | :----------: | :----------: | ------------------------------------------------------------------------------- |
 | #<_ISSUE_NUMBER_> | #<_PR_NUMBER_> |     **<YES/NO>**     | **<YES/NO>** | **<YES/NO>** | #<_RELEASE_ISSUE_>                                                              |
 
-## Why does this issue meet the [cherry-pick criteria](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#cherry-pick-criteria)?
+## Why does this issue meet the [cherry-pick criteria](https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#Cherry-picks)?
```

Comment:
```suggestion
## Why does this issue meet the [cherry-pick criteria](https://go.amp.dev/cherry-picks)?
```

Reply:
Since this is a link to an `.md` file internal to the same Github repo, I don't think we should use go links as they add an extra redirect.

Moreover, I think links of this sort should use relative paths, but that's better done separately as a large-scale cleanup.

Reply:
I would counter that the point of the go links is to have canonical links that can be used across projects (like its current usage in this project's doc as well as AMP.dev), which can then be updated centrally if/when the link changes (which is exactly why this PR/change has to happen this time). The added redirect is minimal imo

Reply:
There are [200+](https://github.com/ampproject/amphtml/search?l=Markdown&q=%22github.com%2Fampproject%2Famphtml%22) direct links and [4](https://github.com/ampproject/amphtml/search?l=Markdown&q=%22go.amp.dev%22) short links in our repo's documentation.

Since this is a widespread thing, we should do a large-scale find-replace for all links that have a go.amp.dev equivalent in a separate PR.

Reply:
Sure; if you file an issue and assign it to me I'll take care of that.

Reply:
Filed #26561.